### PR TITLE
Feat implement refactor suggestions

### PR DESCRIPTION
**Pull request by PineappleIOnic**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: app/views/install/compose.phtml [hunk 1]
```diff
@@ -355,75 +355,6 @@ services:
...
-
-  appwrite-worker-mails:
-    image: <?php echo $organization; ?>/<?php echo $image; ?>:<?php echo $version."\n"; ?>
-    entrypoint: worker-mails
-    container_name: appwrite-worker-mails
-    restart: unless-stopped
-    networks:
-      - appwrite
-    depends_on:
-      - redis
```
Comment (christyjacob4) [inline app/views/install/compose.phtml:412]: why have we removed all this ? 🤔

**Review by christyjacob4 — APPROVED** [review]

File: .env [hunk 2]
```diff
@@ -50,4 +50,4 @@ _APP_MAINTENANCE_RETENTION_AUDIT=1209600
 _APP_USAGE_STATS=enabled
 _APP_LOGGING_PROVIDER=
 _APP_LOGGING_CONFIG=
-OPENRUNTIMES_INACTIVE_THRESHOLD=60
\ No newline at end of file
+_APP_RUNTIMES_INACTIVE_THRESHOLD=60
```
Comment (eldadfux) [inline .env:53]: Let's keep it under the `APP_FUNCTIONS` prefix.

File: app/views/install/compose.phtml [hunk 3]
```diff
@@ -355,52 +355,6 @@ services:
       - _APP_EXECUTOR_SECRET
       - _APP_USAGE_STATS
 
-  appwrite-executor:
```
Comment (eldadfux) [inline app/views/install/compose.phtml:358]: Why are we removing the executor here?

**Review by eldadfux — CHANGES_REQUESTED** [review]

File: app/views/install/compose.phtml (hunk 3 above)
Comment (PineappleIOnic) [reply app/views/install/compose.phtml:358]: There is a second executor higher up in the file: https://github.com/appwrite/appwrite/blob/1db9a872d9803047960aeb80e14b92c6d0f45201/app/views/install/compose.phtml#L159

File: .env (hunk 2 above)
Comment (PineappleIOnic) [reply .env]: Alright, done :+1:

File: .env [hunk 4]
```diff
@@ -50,4 +50,4 @@ _APP_MAINTENANCE_RETENTION_AUDIT=1209600
 _APP_USAGE_STATS=enabled
 _APP_LOGGING_PROVIDER=
 _APP_LOGGING_CONFIG=
-_APP_RUNTIMES_INACTIVE_THRESHOLD=60
\ No newline at end of file
+_APP_FUNCTIONS_INACTIVE_THRESHOLD=60
```
Comment (eldadfux) [inline .env:53]: Let's move it near the functions family  😅

**Review by eldadfux — APPROVED** [review]

**Pull request merged**